INIT.md

```diff
@@ -1,10 +1,54 @@
-## webpack init
+# webpack-cli init
 
-Through [yeoman](http://yeoman.io/), the `webpack-cli init` feature allows people to create scaffolds and generate new projects quickly. An npm dependency that scaffolds a `webpack.config.js` through `webpack-cli` is what we refer to as an **addon**.
+`webpack-cli init` is used to initialize `webpack` projects quickly by scaffolding configuration and installing modules required for the project as per user preferences.
 
-We ask several questions in the default generator to get you started.
+## Initial Setup
+A. **Local setup**
 
----
+Follow given steps to locally setup `webpack-cli init` by installing dependencies:
+1. Create `package.json` through npm
+
+	```shell
+	$ npm init
+	```
+
+2. Install `webpack` and `webpack-cli` as devDependencies
+
+	```shell
+    $ npm install --save-dev webpack webpack-cli
+    ```
+
+3. Install @webpack-cli/init package to add init addon
+
+	```shell
+    $ npm install -D @webpack-cli/init
+    ```
+	
+B. **Global Setup**
+
+Follow following steps to setup `webpack-cli init` globally:
+1. Install `webpack` and `webpack-cli` globally
+	```shell
+	$ sudo npm install -g webpack webpack-cli
+	```
+	
+2. Install @webpack-cli/init package to add init addon
+	```shell
+	$ sudo npm install -g @webpack-cli/init
+	```
+    
+## Usage
+A. **For local setup**:
+```shell
+$ npx webpack-cli init
+```
+
+B. **For global setup**
+```shell
+$ webpack-cli init
+```
+
+### Questions asked by generator
```

Comment:
Nice work. :100: :1st_place_medal: 

How about mentioning which specific property/key the question is related to and also provide a link to webpack docs for every question? Similar to how you did for entry. :)

Reply:
Sure, I will where ever possible in next commit.

Reply:
this has been resolved @dhruvdutt ?

Reply:
Looks good.  Why have we skipped mentioning property for the last question?